intrinsic_sdk_cmake/cmake/api/skill/resource/skill.Dockerfile

```diff
@@ -54,10 +55,11 @@ RUN . /opt/intrinsic/intrinsic_sdk_cmake/install/setup.sh \
         --event-handlers console_direct+ console_stderr- \
         --merge-install \
         --executor=sequential \
-        --packages-up-to $SKILL_PACKAGE
+        --packages-up-to $SKILL_PACKAGE \
+    && which gcc
```

Comment:
What is the `which gcc` needed for here?

Reply:
Vestigial, I'll remove it.